README.md

```diff
@@ -308,8 +308,70 @@ just added.
 ## API
 
 <generated_api_start />
+#### The Jest global environment
```

Comment:
So many of these functions don't have specific types, is the thing. "string" and "number" flow types are pretty intuitive to people, but any / mixed / the union stuff are fairly complicated if you don't know flow (as most of our audience doesn't). For now let's just describe the types in the text.